website/source/guides/advanced-scheduling/affinity.html.md

```diff
@@ -0,0 +1,217 @@
+       port_map {
+         db = 6379
+       }
+     }
+
+     resources {
+       network {
+         port "db" {}
+       }
+     }
+
+     service {
+       name = "redis-cache"
+       port = "db"
+       check {
+         name     = "alive"
+         type     = "tcp"
+         interval = "10s"
+         timeout  = "2s"
+       }
+     }
+   }
+ }
+}
+```
+Note that we used the `affinity` stanza and specified `dc2` as the
+value for the [attribute][attributes] `${node.datacenter}`. We used the value `100` for the [weight][weight] which will cause the Nomad schedular to rank nodes in datacenter `dc2` with a higher score. Keep in mind that weights can range from -100 to 100, inclusive. Negative weights serve as anti-affinities which cause Nomad to avoid placing allocations on nodes that match the criteria.
+
+### Step 3: Register the Job `redis.nomad`
+
+Run the Nomad job with the following command:
+
+```shell
+$ nomad run redis.nomad 
+==> Monitoring evaluation "11388ef2"
+    Evaluation triggered by job "redis"
+    Allocation "0dfcf0ba" created: node "6b6e9518", group "cache1"
+    Allocation "89a9aae9" created: node "3592943e", group "cache1"
+    Allocation "9a00f742" created: node "6b6e9518", group "cache1"
+    Allocation "fc0f21bc" created: node "3dea0188", group "cache1"
+    Evaluation status changed: "pending" -> "complete"
+==> Evaluation "11388ef2" finished with status "complete"
+```
+
+Note that two of the allocations in this example have been placed on node `6b6e9518`. This is the node we configured to be in datacenter `dc2`. The Nomad scheduler selected this node because of the affinity we specified. All of the allocations have not been placed on this node because the Nomad scheduler considers other factors in the scoring such as bin packing. This helps avoid placing too many instances of the same job on a node and prevents reduced capacity during a node level failure. We will take a detailed look at the scoring in the next few steps.
+
+### Step 4: Check the Status of the `redis` Job
+
+At this point, we are going to check the status of our job and verify where our
+allocations have been placed. Run the following command:
+
+```shell
+$ nomad status redis
+```
+
+You should see 4 instances of your job running in the `Summary` section of the
+output as show below:
+
+```shell
+...
+Summary
+Task Group  Queued  Starting  Running  Failed  Complete  Lost
+cache1      0       0         4        0       0         0
+
+Allocations
+ID        Node ID   Task Group  Version  Desired  Status   Created    Modified
+0dfcf0ba  6b6e9518  cache1      0        run      running  1h44m ago  1h44m ago
+89a9aae9  3592943e  cache1      0        run      running  1h44m ago  1h44m ago
+9a00f742  6b6e9518  cache1      0        run      running  1h44m ago  1h44m ago
+fc0f21bc  3dea0188  cache1      0        run      running  1h44m ago  1h44m ago
+```
+
+As stated earlier, you can cross-check this output with the results of the
```

Comment:
Could we remove this "As stated earlier" phrase from here and everywhere else, it makes the sentences longer and doesn't add much value.